system/Database/BasePreparedQuery.php

```diff
@@ -125,8 +126,13 @@ public function execute(...$data)
         try {
             $exception = null;
             $result    = $this->_execute($data);
+        } catch (BadMethodCallException $exception) {
+            throw $exception;
```

Comment:
When SQLite3 will throw `BadMethodCallException`?

Reply:
https://github.com/codeigniter4/CodeIgniter4/blob/fff45081216ff025bd5e6138ae33338c65eb33d3/system/Database/SQLite3/PreparedQuery.php#L65-L69
But I changed the code.

Reply:
Oh... that makes sense. Thank you.